src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -72,6 +73,7 @@ public function preAuthorizePayment($idPayment);
 
     /**
      * Specification:
+     * - Distributes the prices for PayoneCaptureTransfer.order items and expenses depending on the payment method.
```

Comment:
```suggestion
     * - Distributes the prices for `PayoneCaptureTransfer.order` items and expenses depending on the payment method.
```

Please, fix similar cases below.

Reply:
Do we have any RFC for this? Because mostly we use a property name without quotes (33 vs 100+).